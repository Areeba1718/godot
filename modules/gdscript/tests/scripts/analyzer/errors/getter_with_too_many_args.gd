var a: int: get = get_named

func test():
	pass

func get_named(name: int, oops: int) -> int:
	prints("get_named", name)
	return a
