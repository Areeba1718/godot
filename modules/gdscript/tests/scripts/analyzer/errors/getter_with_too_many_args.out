GDTEST_ANALYZER_ERROR
A getter cannot have more than 1 mandatory argument.
